src/include/daos_obj_class.h

```diff
@@ -60,17 +60,23 @@ enum daos_obj_redun {
 	 * Examples:
 	 * - 2P1: 2+1 EC object
 	 * - 4P2: 4+2 EC object
+	 * - 4P3: 4+3 EC object
 	 * - 8P2: 8+2 EC object
+	 * - 8P3: 8+3 EC object
 	 * - 16P2: 16+2 EC object
+	 * - 16P3: 16+3 EC object
 	 */
 	OR_RS_2P1 = 32,
 	OR_RS_2P2,
 	OR_RS_4P1,
 	OR_RS_4P2,
+	OR_RS_4P3,
```

Comment:
cannot add it here to avoid break compatibility, as cannot change the original value of previously defined redun.
so need to select a place that will not change original defined value.